pkg/envoy/sort.go

```diff
@@ -83,51 +83,52 @@ type PortNetworkPolicyRuleSlice []*cilium.PortNetworkPolicyRule
 
 // PortNetworkPolicyRuleLess reports whether the r1 rule should sort before
 // the r2 rule.
+// L3-L4-only rules are less than L7 rules.
 func PortNetworkPolicyRuleLess(r1, r2 *cilium.PortNetworkPolicyRule) bool {
-	remotePolicies1, remotePolicies2 := r1.RemotePolicies, r2.RemotePolicies
+	// TODO: Support Kafka.
```

Comment:
Can we raise a GH issue for Kafka  and put reference here instead of a TODO only?

Reply:
Don’t worry, that won’t fall through the cracks.